k8s/network.go

```diff
@@ -120,10 +111,14 @@ func newNetwork(params networkParams) (network.Network, error) {
 
 // NewNetwork returns a new network whose initial state is specified in the config
 func NewNetwork(conf network.Config, log logging.Logger) (network.Network, error) {
+	k8sClient, err := newK8sClient()
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create k8s client: %w", err)
```

Comment:
Didn't we say we use `%s` for error messages formatting

Reply:
Inside of `log.Info`, `log.Debug`, etc., yes, because those methods don't work with the `%w` directive.

In `fmt.Errorf` it doesn't _really_ matter if we do `%` or `%s` for errors. `%w` "wraps" the error but we never use that functionality here: https://go.dev/blog/go1.13-errors